### PR TITLE
[4.0] com_languages sidebar

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the work to remove the sidebar submenu. This pr removes it from Languages, Content Languages and Overrides.

### CODE REVIEW

File: administrator/components/com_languages/View/Languages/HtmlView.php [hunk 1]
```diff
@@ -151,8 +140,6 @@ protected function addToolbar()
 
 		ToolbarHelper::help('JHELP_EXTENSIONS_LANGUAGE_MANAGER_CONTENT');
```
Comment (Quy) [inline administrator/components/com_languages/View/Languages/HtmlView.php]:
```suggestion
```

**Comment by Quy** [comment]
Remove `use Joomla\Component\Languages\Administrator\Helper\LanguagesHelper;`

File: administrator/components/com_languages/View/Overrides/HtmlView.php [hunk 2]
```diff
@@ -132,8 +122,5 @@ protected function addToolbar()
 		ToolbarHelper::divider();
 		ToolbarHelper::help('JHELP_EXTENSIONS_LANGUAGE_MANAGER_OVERRIDES');
```
Comment (Quy) [inline administrator/components/com_languages/View/Overrides/HtmlView.php]:
```suggestion
```

**Comment by brianteeman** [comment]
> Remove use Joomla\Component\Languages\Administrator\Helper\LanguagesHelper;

From where? That doesn't exist as far as I can see

**Comment by Quy** [comment]
In `HtmlView.php` files at the top in the `use` area.

```
use Joomla\CMS\Helper\ContentHelper;
use Joomla\CMS\Language\Text;
use Joomla\CMS\MVC\View\HtmlView as BaseHtmlView;
use Joomla\CMS\Toolbar\Toolbar;
use Joomla\CMS\Toolbar\ToolbarHelper;
use Joomla\Component\Languages\Administrator\Helper\LanguagesHelper;
```

**Comment by brianteeman** [comment]
I see it now but I think other functions in that helper are being used

**Comment by Quy** [comment]
Searched the 3 files and it was only used for the `addSubmenu`.

**Comment by sanderpotjer** [comment]
@brianteeman I'm using PhpStorm and that is displaying if something is used:
Screenshot 2019-05-04 at 13 08 41

Light grey means not in use, so `LanguagesHelper` can be cleaned up (for all 3 `HtmlView.php` files).

**Comment by brianteeman** [comment]
Yeah I will fix it this afternoon

**Comment by brianteeman** [comment]
Updated as requested

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on d978f0e0a3e7b311cac5c91d3c1ad57106946b14<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24774">issues.joomla.org/tracker/joomla-cms/24774</a>.</sub>

File: administrator/components/com_languages/Helper/LanguagesHelper.php [hunk 3]
```diff
@@ -21,32 +21,6 @@
...
-	 * Configure the Linkbar.
-	 *
-	 * @param   string  $vName  The name of the active view.
-	 *
-	 * @return  void
-	 */
-	public static function addSubmenu($vName)
-	{
-		\JHtmlSidebar::addEntry(
-			Text::_('COM_LANGUAGES_SUBMENU_INSTALLED'),
```
Comment (Quy) [inline administrator/components/com_languages/Helper/LanguagesHelper.php:34]: Should these language strings be removed?

Comment (infograf768) [reply administrator/components/com_languages/Helper/LanguagesHelper.php:34]: All depends on the final decision concerning the display of the System stuff.
I still hope we can get direct access via the System Menu to all stuff there (and not be forced to load the entire system page) OR at least, when Custom Admin Menus are corrected,  be able to add a specific custom menu item for each of these as to other such former sidebar submenus.

Comment (infograf768) [reply administrator/components/com_languages/Helper/LanguagesHelper.php:34]: Hmm... My mistake.
In 3.x we have different constants for these, for example:
`MOD_MENU_COM_LANGUAGES_SUBMENU_INSTALLED`
`MOD_MENU_COM_LANGUAGES_SUBMENU_CONTENT`
`MOD_MENU_COM_LANGUAGES_SUBMENU_OVERRIDES`
These are still present in the mod_menu.ini file in 4.x

Therefore I guess you are right and `COM_LANGUAGES_SUBMENU_XXX` can be deleted.

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on 8116fa3e8963bfefe1de54130df822479a37ec10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24774">issues.joomla.org/tracker/joomla-cms/24774</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 8116fa3e8963bfefe1de54130df822479a37ec10<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24774">issues.joomla.org/tracker/joomla-cms/24774</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24774">issues.joomla.org/tracker/joomla-cms/24774</a>.</sub>

**Comment by SniperSister** [comment]
Merged, Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**